index.html

```diff
@@ -228,14 +228,31 @@ <h1 class="contact-title">
                </p>
             </div>
 
-            <div class="contact-form">
-               <input type="text" placeholder="Text Input" class="user-text">
-               <input type="text" name="email" id="email" placeholder="kibirigejohn005@gmail.com" class="user-email">
-               <textarea name="text-area" id="user-text" cols="30" rows="10" class="user-text-area" placeholder="Enter text here"></textarea>
-               <button class="see-projects project-btn">
-                  See Project
-               </button>
-            </div>
+            <form class="contact-form" action="https://formspree.io/f/xnqwkjoy" method="post">
+               <ul>
+                  <li>
+                     <label for="user-name"></label>
+                     <input type="text" placeholder="Text Input" class="user-name" id="user-name" name="user-name" maxlength="30" required >
+                  </li>
+                  <li>
+                     <label for="email"></label>
+                     <input type="email" name="email" id="email" placeholder="kibirigejohn005@gmail.com" class="user-email" required >
+
+                  </li>
+                  <li>
+                     <label for="user-text"></label>
+                     <textarea name="text-area" id="user-text" cols="30" rows="10" class="user-text-area" placeholder="Enter text here" maxlength="500" required></textarea>
```

Comment:
Please provide appropriate placeholders (e.g. `Enter your name here`) to improve UX. 

<img width="392" alt="Screen Shot 2022-07-05 at 13 31 00" src="https://user-images.githubusercontent.com/82426682/177392647-a0acdb56-05e6-4fe7-ad06-365419d4335e.png">